test/e2e/cypress/integration/profiles/webhoooks/registration/success.spec.ts

```diff
@@ -147,7 +147,9 @@ context("Registration success with email profile with webhooks", () => {
           expect(identity.schema_id).to.equal("default")
           expect(identity.schema_url).to.equal(`${APP_URL}/schemas/ZGVmYXVsdA`)
           expect(identity.traits.email).to.equal("updated-" + email)
-          expect(identity.metadata_public.some).to.equal("public fields")
+          expect((identity as any).metadata_public.some).to.equal(
```

Comment:
These types still use a pretty old version of `@ory/kratos-client`, where `metadata_public` didn't exist, yet. But I want to do that in a future PR.